### PR TITLE
Issue 1 Create an Entry - Alvaro Diaz

### PR DESCRIPTION
**Pull request by alvarodiazjr**
Screen Shot 2022-07-19 at 2 32 07 PM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,13 +14,56 @@
...
 </head>
-<body>
-  <header>
-  </header>
-  <main>
-  </main>
-  <script src="js/data.js"></script>
-  <script src="js/main.js"></script>
+<body class="gray-background">
+  <div data-view = "entry-form">
```
Comment (codingcodymiller) [inline index.html]: No spaces around equal signs in HTML, please

File: css/layout.css [hunk 2]
```diff
@@ -3,11 +3,30 @@
 }
 
 .container {
+  font-family: 'Open Sans', sans-serif;
```
Comment (codingcodymiller) [inline css/layout.css]: Don't pollute layout classes, make this for a separate class.

File: css/layout.css [hunk 3]
```diff
@@ -3,11 +3,30 @@
...
 }
 
+.header-container {
+  font-family: 'Open Sans', sans-serif;
+  max-width: 575px;
+  margin: 0 auto;
+  padding: 0 1.5rem;
+  color: white;
+  background-color: rgb(105, 37, 105);
+}
```
Comment (codingcodymiller) [inline css/layout.css]: Don't make variants of layout classes, you should just be using the `container` class and making an additional class you can add for any styling you need beyond what `container` sets

**Comment by alvarodiazjr** [comment]
Screen Shot 2022-07-19 at 2 44 25 PM

https://user-images.githubusercontent.com/103209321/179856624-1ce29094-1876-46aa-bd38-c1bbf590ccd5.mov


https://user-images.githubusercontent.com/103209321/179856643-0f9417fb-cef5-44bd-9fd7-98c960054fde.mov

revised images and form in action.

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  event.preventDefault();
+
+  var nextEntry = {
+    title: $form.elements.title.value,
+    photoUrl: $form.elements.photo.value,
+    notes: $form.elements.notes.value,
+    nextEntryId: data.nextEntryId
+  };
+
+  data.entries.push(nextEntry);
```
Comment (codingcodymiller) [inline js/main.js]: You should be adding this to the beginning of the array, not the end

File: js/main.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+var $form = document.querySelector('form');
+
+$form.addEventListener('submit', function journalEntry(event) {
+  event.preventDefault();
+
+  var nextEntry = {
+    title: $form.elements.title.value,
+    photoUrl: $form.elements.photo.value,
+    notes: $form.elements.notes.value,
+    nextEntryId: data.nextEntryId
```
Comment (codingcodymiller) [inline js/main.js]: You should name your property `entryId`, not `nextEntryId`, as I discussed with the class yesterday

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**